code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -75,6 +75,9 @@
 		/obj/item/clothing/shoes/brown,
 		/obj/item/clothing/shoes/black,
 	)
+	if(istype(loc.loc,/area/lawoffice))
```

Comment:
This is just awful. The proper way to do this is to create and map a different subtype.